alembic/versions/684501e372d2_adding_oauth.py

```diff
@@ -0,0 +1,82 @@
+"""
+Adding oauth.
+
+Revision ID: 684501e372d2
+Revises: 65167edbb0de
+Create Date: 2023-05-17 20:39:21.658338
+
+"""
+import fastapi_users_db_sqlalchemy as fastapi_users_db_sqlalchemy
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "684501e372d2"
+down_revision = "65167edbb0de"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    """Upgrade the schema to this version."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "user",
+        sa.Column("id", fastapi_users_db_sqlalchemy.generics.GUID(), nullable=False),
+        sa.Column("email", sa.String(length=320), nullable=False),
+        sa.Column("hashed_password", sa.String(length=1024), nullable=False),
+        sa.Column("is_active", sa.Boolean(), nullable=False),
+        sa.Column("is_superuser", sa.Boolean(), nullable=False),
+        sa.Column("is_verified", sa.Boolean(), nullable=False),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_index(op.f("ix_user_email"), "user", ["email"], unique=True)
```

Comment:
Why does email need to be in an array?

Reply:
These alembic files are code-generated and not hand written! [Alembic](https://alembic.sqlalchemy.org/en/latest/) is a tool that automatically constructs DB migrations based on changes to the SQLAlchemy models you define

Reply:
Dude, this is the future. I have to manually write all my SQL migrations at work, since we use Flyway for schema migrations and Hibernate / JPA for our ORM. Cool stuff.